### PR TITLE
GitHub Actions Billing returns total_paid_minutes_used as float64

### LINKED ISSUE #2283: GitHub Actions Billing returns total_paid_minutes_used as float64
GitHub returns the `total_paid_minutes_used` value as float64 in the `/orgs/{org}/settings/billing/actions` endpoint.
Currently an Int is expected.

## Example
```bash
curl -H "Authorization: Bearer <TOKEN>" \
-H "Accept: application/vnd.github.v3+json"  \
https://api.github.com/orgs/{org}/settings/billing/actions
```
```json
{
  "total_minutes_used": 1234,
  "total_paid_minutes_used": 3959.0,
  "included_minutes": 3000,
  "minutes_used_breakdown": {
    "UBUNTU": 1234,
    "MACOS": 1234,
    "WINDOWS": 0
  }
}
```

### PR DESCRIPTION
**Pull request by raynigon**
Fixes #2283 

GitHub Actions returns total_paid_minutes_used as floating point value.
The Implementation should accept this floating point value as float64.

## Example
```bash
curl -H "Authorization: Bearer <TOKEN>" \
-H "Accept: application/vnd.github.v3+json"  \
https://api.github.com/orgs/{org}/settings/billing/actions
```
```json
{
  "total_minutes_used": 1234,
  "total_paid_minutes_used": 3959.0,
  "included_minutes": 3000,
  "minutes_used_breakdown": {
    "UBUNTU": 1234,
    "MACOS": 1234,
    "WINDOWS": 1234
  }
}
```

### CODE REVIEW

**Review by gmlewis — APPROVED** [review]
Thank you, @raynigon !
LGTM.

Awaiting second LGTM/Approval from any other contributor to this repo before merging.

**Comment by gmlewis** [comment]
I'm driving several hundred miles today... I may be slow to respond.

**Comment by raynigon** [comment]
@Parker77 - do you have time to review this PR?

**Review by Parker77 — APPROVED** [review]
LGTM!

**Comment by gmlewis** [comment]
Thank you, @Parker77 !
Merging.

**Pull request merged**
